hack/jenkins/e2e.sh

```diff
@@ -231,13 +231,19 @@ echo "Test Environment:"
 printenv | sort
 echo "--------------------------------------------------------------------------------"
 
-if [[ "${E2E_UP,,}" == "true" ]]; then
+# We get the kuberntes tarballs on either cluster creation or when we want to
+# replace existing ones in a multi-step job (e.g. a cluster upgrade).
+if [[ "${E2E_UP,,}" == "true" || ${JENKINS_FORCE_GET_TARS:-} =~ ^[yY]$ ]]; then
```

Comment:
We use a `=~` comparison about 5 other times in this file and never quote so I copied that style. Do you want me to go and quote all of them?